servers/lib/src/main.ts

```diff
@@ -13,15 +14,19 @@ program
     'The lib microservice is responsible for handling and serving the contents of library assets of the DTaaS platform. It provides API endpoints for clients to query, and fetch these assets.',
   )
   .option('-c, --config <path>', 'set the config path (default .env)')
+  .option(
+      '-f --cloudcmd <configPath>',
+      'enable the file server with the specified config',
+    )
   .helpOption('-h, --help', 'display help for libms')
   .showHelpAfterError();
 
 program.parse(process.argv);
 
 const options: ProgramOptions = program.opts();
 
-if (options.config) {
-  bootstrap({ config: options.config, runHelp: () => program.help() });
-} else {
-  bootstrap({ runHelp: () => program.help() });
-}
+bootstrap({
```

Comment:
Does this code fragment has the same behaviour as the one it is replacing?

Reply:
Yes, it's passing all the parameters into bootstrap, so if the parameters aren't set, they will be undefined.